app/app/src/main/java/org/fastchat/MainActivity.java

```diff
@@ -0,0 +1,38 @@
+package org.fastchat;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.widget.Button;
+
+import org.fastchat.screens.ChatActivity;
+
+public class MainActivity extends AppCompatActivity {
+
+    // Declare elements here
+    Button btnGoToChatActivity;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        btnGoToChatActivity = findViewById(R.id.btnGoToChatActivity);
+
+        btnGoToChatActivity.setOnClickListener(v -> {
+
+            String ip = ""; // IP WebSocket server
```

Comment:
mmm, how can I set the IP without a change in the logic?

Reply:
It will be implemented in the next PR